code/modules/GOAP/monkey/info_provider.dm

```diff
@@ -0,0 +1,74 @@
+/datum/goap_info_provider/monkey/GetWorldState(datum/goap_agent/agent)
+	. = list()
+	.["GUN"] = FALSE
```

Comment:
why not just do these as part of the `. = list()` since they're static

Reply:
To be honest I like this format more as it’s personally easier to keep the track of values and you couldn’t fit everything in the list as some values need checks anyway, if line count is an issue I can fix it then though

Reply:
It saves time to initialize the list with values.

Reply:
> you couldn’t fit everything in the list as some values need checks

It's a list of bools, you can just do the checks and assign them to variables, then use those in the big list() call.

Reply:
i'm just suggesting you init the list with whats in lines 3-9

Reply:
that makes no sense,  then you might as well do what he's doing already you chumbis.

Anyways why on't you just make the static entries in the initial list() call, then any dynamic ones can just go below it?